File: auth-api/src/main/java/net/plshark/ObjectNotFoundException.java
package net.plshark;

/**
 * Exception that indicates an object was not found
 */
public class ObjectNotFoundException extends Exception {

    private static final long serialVersionUID = -8213485241336803509L;

    /**
     * Create a new instance
     * @param message the detail message
     */
    public ObjectNotFoundException(String message) {
        super(message);
    }

    /**
     * Create a new instance
     * @param message the detail message
     * @param cause the cause
     */
    public ObjectNotFoundException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: user-error-api/src/main/kotlin/net/plshark/errors/ObjectNotFoundException.kt
package net.plshark.errors

/**
 * Exception that indicates an object was not found
 */
class ObjectNotFoundException : RuntimeException {
    /**
     * Create a new instance
     * @param message the detail message
     */
    constructor(message: String?) : super(message)

    /**
     * Create a new instance
     * @param message the detail message
     * @param cause the cause
     */
    constructor(message: String?, cause: Throwable?) : super(message, cause)
}
